components/KiboAppHeader.vue

```diff
@@ -110,8 +118,10 @@ import { useAsync } from "@nuxtjs/composition-api"
 import useUiState from "../composables/useUiState"
 import * as logo from "../assets/kibo_logo.png"
 import { usePurchaseLocation } from "../composables"
```

Comment:
Instead of using .../  use @/composables for consistency purpose

Reply:
solved.